### ISSUE TITLE
Broken entry in unwanted-iranian.txt list

### ISSUE THREAD

**Issue by Leah96xxx** [DRSDavidSoft/additional-hosts#12]
There is a broken entry in domains/blacklist/unwanted-iranian.txt

The broken entry is just "cdn." which results in every domain with the hostname "cdn" getting blocked. I have attached a screenshot of the faulty entry (highlighted in orange) as well as an example false positive from AdGuard Home.
![unknown](https://user-images.githubusercontent.com/88419550/174393759-63afbcde-65e4-4a4f-914a-51e7115dc919.png)
![unknown](https://user-images.githubusercontent.com/88419550/174393789-38136999-5ec4-4f95-a834-1522244c6a6d.png)
![unknown](https://user-images.githubusercontent.com/88419550/174393802-df1ae163-b7b5-4535-94fe-72a2a19cfaed.png)

Edit: I just noticed the pull request

**Comment by Soneydom13** [comment]
I can confirm it causing the same issue with causing facebook images to fail loading as well as tiktok videos

**Comment by DRSDavidSoft** [comment]
I am _very_ sorry for this major mistake guys, have resolved it via #11. 😭 
Thanks for reporting the issue and continuing to use my list(s), I promise a major problem like this will _never_ happen again

I just fixed the problem.  It is obvious that an entry such as `cdn.` is never a good idea to be blocked, I'm not sure why I made the mistake (it should have been `cdn.adivery.com`).

I apologize for any disturbance and inconvenience that this mistake have caused to people pulling in the latest commit.  I deeply appreciate reporting the issue, and thanks for continuing to use my lists!
